cmd/deployment.go

```diff
@@ -165,7 +165,7 @@ func newDeploymentListCmd(client *houston.Client, out io.Writer) *cobra.Command
 }
 
 func newDeploymentUpdateCmd(client *houston.Client, out io.Writer) *cobra.Command {
-	example := ` 
+	example := `
```

Comment:
I don't see diff between before and after 🤔

Reply:
Trailing whitespace was deleted.